src/createAction.js

```diff
@@ -1,22 +1,27 @@
 import identity from 'lodash/identity';
 import isFunction from 'lodash/isFunction';
 import isUndefined from 'lodash/isUndefined';
+import isNull from 'lodash/isNull';
```

Comment:
I'd instead use [`isNil`](https://lodash.com/docs/#isNil), so we check for `null` **OR** `undefined`.